### PR TITLE
Add Jiva-functional  and chaos experiment into rancher-pipeline

### PR DESCRIPTION
**Pull request by shashank855**
Signed-off-by: awesomeshashank855 <ranjanshashank855@gmail.com>

### CODE REVIEW

**Review by nsathyaseelan — APPROVED** [review]
Lgtm

File: stages/functional/Jiva-Snapshot-Clone/jiva-snapshot-clone [hunk 1]
```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+
+pod() {
+sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'cd oep-e2e-rancher && bash stages/functional/Jiva-Snapshot-Clone/jiva-snapshot-clone node '"'$CI_JOB_ID'"'' '"'$CI_PIPELINE_ID'"' '"'$RELEASE_TAG'"'
```
Comment (harshshekhar15) [inline stages/functional/Jiva-Snapshot-Clone/jiva-snapshot-clone:5]: Pass `commit_id` here as expected by node function.

Comment (shashank855) [reply stages/functional/Jiva-Snapshot-Clone/jiva-snapshot-clone:5]: I removed commit ID from node function.

File: stages/chaos/Jiva-App-kill/jiva-app-kill [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+echo "*******Deploying Busybox Application****"
+
+test_name=$(bash utils/generate_test_name testcase=busybox-provision-app-kill-jiva metadata="")
+echo $test_name
+
+################
+# LitmusBook 1 #
+################
+
+cd litmus
```
Comment (nsathyaseelan) [inline stages/chaos/Jiva-App-kill/jiva-app-kill]: @shashank855  in which job this litmus directory is cloned

**Review by nsathyaseelan — CHANGES_REQUESTED** [review]

File: stages/chaos/Jiva-Controller-Network-Delay/ctrl-network-delay [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+}
+
+node() {
+
+export KUBECONFIG=~/.kube/cluster2_config
+
+job_id=$(echo $1)
+pipeline_id=$(echo $2)
+releaseTag=$(echo $3)
+case_id=TSXL
```
Comment (nsathyaseelan) [inline stages/chaos/Jiva-Controller-Network-Delay/ctrl-network-delay:15]: @shashank855  can you remove this case_id

**Review by nsathyaseelan — CHANGES_REQUESTED** [review]

**Review by nsathyaseelan — APPROVED** [review]
lgtm

**Review by harshshekhar15 — APPROVED** [review]

**Pull request merged**